server/api.js

```diff
@@ -3,7 +3,29 @@ import { Router } from "express";
 const router = Router();
```

Comment:
I saw you installed express (line 72 in package.json) , however i don't see you using express route in your server/api.js file 
const express = require("express");
const app = express();
const PORT = process.env.PORT = 3000;

let router = express.Router();

Reply:
Thank you for review, please check the project configurations